tests/functional/test_install.py

```diff
@@ -76,21 +79,18 @@ def test_editable_install(script):
     assert not result.files_updated
 
 
-@pytest.mark.network
-def test_install_editable_from_svn(script, tmpdir):
+def test_install_editable_from_svn(script):
     """
     Test checking out from svn.
     """
+    checkout_path = _create_test_package(script, vcs='svn')
     result = script.pip(
         'install',
-        '-e',
-        '%s#egg=initools-dev' %
-        local_checkout(
-            'svn+http://svn.colorstudy.com/INITools/trunk',
-            tmpdir.join("cache")
-        )
+        '-e', path_to_url(checkout_path) + '#egg=version-pkg'
     )
-    result.assert_installed('INITools', with_files=['.svn'])
+    result.assert_installed(
+        'version-pkg', pkg_dir=Path(checkout_path),
```

Comment:
don't think you should be overriding the pkg_dir here.
previously the install from was from a "svn+file://" location into the virtual environment
it seems you just have an in-place editable install now.